src/providers/tiktok.ts

```diff
@@ -0,0 +1,73 @@
+import { custom } from "openid-client"
+import type { TokenSet, TokenSetParameters } from "openid-client"
+import { Profile } from ".."
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+export interface TikTokProfile extends Profile {
+  avatar: string
+  open_id: string
+  display_name: string
```

Comment:
Anything else that you can decipher from the returned `tikTokData` that could be useful for people here?